regexes/bots.yml

```diff
@@ -500,7 +500,15 @@
   producer:
     name: 'Google Inc.'
     url: 'http://www.google.com'
-
+    
+- regex: 'Seznam Email Proxy'
+  name: 'SeznamEmailProxy'
```

Comment:
Guess you switched the content of `regex` and `name`

Reply:
Mozilla/5.0 SeznamEmailProxy/2.0.174
--


Here is full user agent from my access log.

Reply:
About test - you mean unit test? I never write it :(

Reply:
ok. will add it after merging